src/apps/vpn/translations/strings.yaml

```diff
@@ -155,6 +155,9 @@ splittunnel:
     comment:
       - This displays as informational text below split tunneling settings. The link directs to a sumo article which will help certain users who are having network issues using this feature.
   infoLinkText: Learn more
+  infoCardDescription: 
+    value: VPN must be off to edit App exclusions
```

Comment:
Since you pointed out that this is not an actual menu, do we need the uppercase `A`?